docs/_sass/_docs-home.scss

```diff
@@ -140,6 +140,7 @@
                 border-color: $docs-home-dark-color !important;
                 background-color: transparent !important;
                 color: $docs-home-dark-color !important;
+                width: 100%;
```

Comment:
Necessary for the search box on the documentation website now that `width: 100%` was removed from input element base styles.